src/app/recipes/recipe-list/recipe-list.component.html

```diff
@@ -6,16 +6,7 @@
 <hr>
 <div class="row">
   <div class="col-xs-12">
-    <a href="#" class="list-group-item clearfix" *ngFor="let recipe of recipes;">
-      <div class="pull-left">
-        <h4 class="list-group-item-heading">{{recipe.name}}</h4>
-        <p class="list-group-item-text">{{recipe.description}}</p>
-      </div>
-      <span class="pull-right">
-        <img src="{{recipe.imagePath}}" alt="" class="img-responsive" style="max-height: 50px;">
-      </span>
-    </a>
-    <app-recipe-item></app-recipe-item>
+    <app-recipe-item *ngFor="let recipeElement of recipes;" (click)="displayRecipeDetails(recipeElement)"
```

Comment:
Why list is responsible for click on recipeItem? It seems handling item click should be delegated to recipeItem